test/e2e/performanceprofile/testdata/render-expected-output/default/manual_node.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: config.openshift.io/v1
+kind: Node
+metadata:
```

Comment:
Will the MCO bootstrap be happy about the ownerReference? What if we remove the performance profile later? Will it delete the config as well?

Reply:
This is the test file, but the MCO is happy with the Node object. I would suggest not changing from cgroupv1 to cgroupv2 automatically since workloads may not be compatible with cgroupv2.